### ISSUE TITLE
Data discrepancy

### ISSUE THREAD

**Issue by catmanjan** [catmanjan/anutimetable#118]
> Hi Jan,
> 
> I've been poking around with the ANU Timetable Builder for a couple days and I just noticed an incident where the data on the timetable does not match up with that provided by the university. I'm not sure what the cause of it is yet but thought I'd alert you ASAP.
> 
> I observed that with ECON1101, there are three lessons displaying the info of "Lecture1 / 1" on anutimetable.com where two of these are actually supposed to read "Lecture2 / 2" and "Lecture3 / 2" respectively, according to the data at http://timetabling.anu.edu.au/sws2017/
> 
> I figure that it's probable that similar discrepancies exist for other courses.
> 
> The main problem with this is that it entirely throws a spanner in the 'choosing' of lessons because these lessons are inaccurately grouped together.
> 
> I noticed that when iterating through every element in timetableData["ECON1101"] and logging each lesson info, the dud "Lecture1 / 1" lessons were not even included in the data set (ie. only the two Monday lectures, one Tuesday lecture, and Thursday morning lectures were logged). I don't fully understand the implementation of the timetable or how they could be missing from timetableData["ECON1101"] and yet still displayed on the timetable so perhaps this part of my investigation is a red herring.
> 
> tl;dr I've attached an image of the discrepancy
> 
> Cheers,

![discrepant data](https://cloud.githubusercontent.com/assets/137350/22232078/5296b7cc-e23b-11e6-8639-8c29ab78c25f.png)

**Comment by samyex6** [comment]
Fixed: #119
